tests/test_time.py

```diff
@@ -43,9 +43,116 @@ def test_out_of_range(self):
     def test_iso(self):
         assert Time("01:02:03.000004") == Time(1, 2, 3, nanosecond=4_000)
 
+    def test_leap_seconds_parsing(self):
+        # Leap second (60) should be parsed and normalized to 59
+        assert Time("01:02:60") == Time(1, 2, 59)
```

Comment:
Let's move these test cases to the parametrized `TestParseIso.test_valid` method. You can add a comment that they're leap second cases